Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12.16.1-alpine as builder
```

Comment:
We also need to re-align this dockerfile.

Reply:
done